tasks/RedHat/generic/assert-tuned.yml

```diff
@@ -6,6 +6,7 @@
 - name: Report tuned version
   debug:
     msg: "INFO: The installed version of package tuned is: {{ ansible_facts.packages['tuned'][0].version }}"
+  ignore_errors: yes
```

Comment:
Under what conditions will this task emit an error that you need to ignore?

Reply:
If package tuned is not install, the role will fail in this task with a fatal error. With ignore_errors, I can avoid this.

Reply:
You could also do the following:
```yaml
 - name: Report tuned version
   debug:
     msg: "INFO: The installed version of package tuned is: {{ ansible_facts.packages['tuned'][0].version }}"
   when: 'tuned' in ansible_facts.packages
```
In general it is better to avoid having to use `ignore_errors: true` which can sometimes mask unexpected errors.  However, in your case, I don't know if it really matters.